scripts/test.js

```
@@ -1,16 +1,63 @@
-var ESLint = require('eslint').CLIEngine;
-var Jasmine = require('jasmine');
-var path = require('path');
+const ESLint = require('eslint').CLIEngine;
+const Jasmine = require('jasmine');
+const path = require('path');
+const glob = require('glob');
+const fs = require('fs');
 
-var linter = new ESLint({
+// ====
+// LINTER
+// ====
+const linter = new ESLint({
     configFile: path.join(__dirname, '..', '.eslintrc.json')
 });
-var linterReport = linter.executeOnFiles(['index.js', 'src/**/*.js', 'test/**/*.js', 'examples/**/*.js']);
+const linterReport = linter.executeOnFiles(['index.js', 'src/**/*.js', 'test/**/*.js', 'examples/**/*.js']);
 if (linterReport.warningCount > 0 || linterReport.errorCount > 0) {
     console.log(linter.getFormatter()(linterReport.results));
     throw new Error('ESLint errors');
 }
 
+const required = (pathname) => path.resolve(__dirname, '../required', pathname || '');
+const root = (pathname) => path.resolve(process.cwd(), pathname);
+
+// ====
+// REQUIRED FILES CHECK
+// ====
+const globOptions = {
+    cwd: required(),
+    dot: true
+};
+
+glob('*', globOptions, (err, files) => {
+    if (err) {
+        throw new Error('Required files check failed');
```

Don't swallow the error, it makes debugging a PITA. Maybe just concatenate it onto the error message?